### PR TITLE
rocAL - fix bug in the usage of GetImageName

### PR DESCRIPTION
**Pull request by rrawther**
fix unittests which was using incorrect use of GetImageName

### CODE REVIEW

File: utilities/rocAL/rocAL_basic_test/rocAL_basic_test.cpp [hunk 1]
```diff
@@ -50,7 +50,7 @@ int main(int argc, const char ** argv)
     // check command-line usage
     const int MIN_ARG_COUNT = 2;
     if(argc < MIN_ARG_COUNT) {
-        std::cout <<  "Usage: image_augmentation <image_dataset_folder> <label_text_file_path> <test_case:0/1> <processing_device=1/cpu=0>  decode_width decode_height <gray_scale:0/rgb:1> decode_shard_counts \n";
+        std::cout <<  "Usage: ro_basic_test <image_dataset_folder> <label_text_file_path> <test_case:0/1> <processing_device=1/cpu=0>  decode_width decode_height <gray_scale:0/rgb:1> decode_shard_counts \n";
```
Comment (LakshmiKumar23) [inline utilities/rocAL/rocAL_basic_test/rocAL_basic_test.cpp:53]: @rrawther did you mean ./rocal_basic_test in usage

Comment (rrawther) [reply utilities/rocAL/rocAL_basic_test/rocAL_basic_test.cpp]: fixed. I have forgotten to push renamed file

**Comment by rrawther** [comment]
@LakshmiKumar23: Can we merge this PR. I am not able to see codacy warnings though

**Comment by LakshmiKumar23** [comment]
> @LakshmiKumar23: Can we merge this PR. I am not able to see codacy warnings though

@rrawther @hansely I know you were still seeing some issue with the PR?

**Comment by hansely** [comment]
> > @LakshmiKumar23: Can we merge this PR. I am not able to see codacy warnings though
> 
> @hansely I know you were still seeing some issue with the PR? @rrawther Waiting for Hansel to approve this PR

@rrawther Hi Rajy, I'm still seeing the issue with the image file name. 
I commented on the issue #952 about the details. Please take a look and let me know.

https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/issues/952

**Comment by hansely** [comment]
@rrawther @LakshmiKumar23 
I have rebuilt the whole thing and it is working as intended. 
The PR can now be merged.

**Comment by LakshmiKumar23** [comment]
> @rrawther @LakshmiKumar23 I have rebuilt the whole thing and it is working as intended. The PR can now be merged.

@kiritigowda

**Review by LakshmiKumar23 — APPROVED** [review]

**Pull request merged**